resources/user-agents/apps/microsoft/bing-search-app.json

```diff
@@ -0,0 +1,201 @@
+            "Archos 50f Helium": "Archos 50f Helium",
+            "B5060": "Lenovo B5060",
+            "C6603": "Sony C6603",
+            "D5503": "Sony D5503",
+            "D5803": "Sony D5803",
+            "D6503": "Sony D6503",
+            "D6603": "Sony D6603",
+            "E2303": "Sony E2303",
+            "GT-I9195": "Samsung GT-I9195",
+            "GT-I9505": "Samsung GT-I9505",
+            "GT-I9515": "Samsung GT-I9515",
+            "GT-P5210*": "Samsung GT-P5210",
+            "HTC One_M8*": "HTC M8",
+            "HTC One": "HTC M7",
+            "HUAWEI CAN-L11": "Huawei CAN-L11",
+            "HUAWEI GRA-L09": "Huawei GRA-L09",
+            "HUAWEI M2-A01W": "Huawei M2-A01W",
+            "HUAWEI TAG-L21": "Huawei TAG-L21",
+            "HUAWEI VNS-L31": "Huawei VNS-L31",
+            "JERRY": "Wiko Jerry",
+            "KIW-L21": "Huawei KIW-L21",
+            "K00Z": "Asus ME175CG",
+            "LENNY": "Wiko Lenny",
+            "Lenovo A616": "Lenovo A616",
+            "LG-D620": "LG D620",
+            "LG-H818": "LG H818",
+            "LIFETAB_P831X.2": "Medion LifeTab P831X.2",
+            "LIFETAB_P970X": "Lenovo LifeTab P970X",
+            "NEM-L51": "Huawei NEM-L51",
+            "P1040X": "Medion P1040X",
+            "SM-A300FU": "Samsung SM-A300FU",
+            "SM-A310F": "Samsung SM-A310F",
+            "SM-A500FU*": "Samsung SM-A500FU",
+            "SM-A510F": "Samsung SM-A510F",
+            "SM-G531F": "Samsung SM-G531F",
+            "SM-G550T": "Samsung SM-G550T",
+            "SM-G800F": "Samsung SM-G800F",
+            "SM-G850F*": "Samsung SM-G850F",
+            "SM-G900F": "Samsung SM-G900F",
+            "SM-G903F": "Samsung SM-G903F",
+            "SM-G925F*": "Samsung SM-G925F",
+            "SM-G930A": "Samsung SM-G930A",
+            "SM-G930F": "Samsung SM-G930F",
+            "SM-G935F": "Samsung SM-G935F",
+            "SM-J320F": "Samsung SM-J320F",
+            "SM-J510FN": "Samsung SM-J510FN",
+            "SM-N910V": "Samsung SM-N910V",
+            "SM-N920P": "Samsung SM-N920P",
+            "SM-N920V": "Samsung SM-N920V",
+            "SM-T235": "Samsung SM-T235",
+            "SM-T335": "Samsung SM-T335",
+            "SM-T525": "Samsung SM-T525",
+            "SM-T535": "Samsung SM-T535",
+            "SM-T555": "Samsung SM-T555",
+            "Tab2A7-10*": "Lenovo Tab 2 A7-10",
+            "VIE-L09": "Huawei VIE-L09",
+            "VK810 4G": "LG VK810",
+            "XT1068": "Motorola XT1068",
+            "ZTE Blade L5": "ZTE Blade L5"
+          },
+          "platforms": [
+            "Android_7_0",
+            "Android_6_0",
+            "Android_5_1", "Android_5_0",
+            "Android_4_4", "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0"
+          ]
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Chrome/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
```

Comment:
You didn't add the "mobile-less" tablet pattern in this file, any particular reason, not seeing any UAs without it?

Reply:
I don't have a UA for this case.